test/hemera/metadata.spec.js

```diff
@@ -97,4 +97,59 @@ describe('Metadata', function() {
       )
     })
   })
+
+  it('Should be able to update metadata', function(done) {
```

Comment:
Could you name it like "Should set metadata context correctly when child call use the same property"